### ISSUE TITLE
Write e2e tests

### ISSUE THREAD

**Issue by jamesgeorge007** [jamesgeorge007/scaffold-static#11]
[ ] `scaffold-static -V` || `scaffold-static --version`
[ ] `scaffold-static -h` || `scaffold-static --help`
[ ] `scaffold-static -i` [unknown option]
[ ] `scaffold-static cmd` [unknown command]

**Comment by jamesgeorge007** [comment]
@ivaibhavjindal wanna give it a try here?

**Comment by ivaibhavjindal** [comment]
@jamesgeorge007 Yeah, would love to but I don't have a prior experience of writing tests. So could you suggest how do I start with this?

**Comment by jamesgeorge007** [comment]
> @jamesgeorge007 Yeah, would love to but I don't have a prior experience of writing tests. So could you suggest how do I start with this?

DM me on the Discord channel of JWOC.
